xml/sle_update_preparation.xml

```diff
@@ -66,23 +66,23 @@
   </para>
 
   <para>
-   Check the release notes to see whether:
+   Check the release notes to check the following.
```

Comment:
If you change this, you also have to change the bullet points. "Check whether * your hardware needs…" is a complete sentence, so are the others. After the change, they no longer work.

Additionally, the bullet points are becoming declarative sentences, means the hardware *does* need special considerations, software packages *have* changed significantly, and so on. We don't know if that is true, users have to check.

Reply:
> Check the release notes to see whether:

Breaking a sentence to introduce a list is a very bad practice.